src/guide/extras/demos/Colors.vue

```diff
@@ -11,7 +11,7 @@ function onMousemove(e) {
     :style="{ backgroundColor: `hsl(${x}, 80%, 50%)` }"
     class="demo movearea"
   >
-    <p>Move your mouse across this div...</p>
+    <p>移动鼠标横着穿过这个 div...</p>
```

Comment:
```suggestion
    <p>Move your mouse across this div...</p>
```

see https://github.com/vuejs-translations/docs-zh-cn/issues/35